tendrl/commons/jobs/__init__.py

```diff
@@ -189,7 +189,6 @@ def process_job(jid):
                 pass
             lock_info = dict(node_id=NS.node_context.node_id,
```

Comment:
I think keeping this as dict would be better than making it as a dict. Dictionary comparison works perfectly, it was the list that was the issue.